test-train/src/genius/Main.java

```diff
@@ -0,0 +1,24 @@
+package genius;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args){
+        Scanner scan = new Scanner(System.in);
+
+        int a = scan.nextInt();
```

Comment:
Please rename this variable to firstNumber and secondNumber
And rename s to result

Reply:
done